css/style.css

```diff
@@ -1,18 +1,522 @@
-.partners-list {
-	width: 100%;
-	margin-bottom: 20px;
-	border: 1px solid #000000;
-	border-collapse: collapse; 
-	background-color: #7de0d3;
+
+body{
+	font-family: "Bakbak One", "Roboto", sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	color: #fff;
+	background-color: #1E1E1E;
+}
+
+a{
+  text-decoration: none;
+  color: currentColor;
+  text-transform: uppercase;
+}
+
+ul,
+ol{
+  list-style: none;
+  color: currentColor;
+}
+
+h1,
+h2,
+h3{
+  color: currentColor;
+  text-transform:uppercase;
+}
+
+h1{
+  font-size: 72px;
+}
+
+h2{
+  font-size: 44px;
+  line-height:56px;
+}
+
+h3{
+  font-size: 24px;
+}
+
+header{
```

Comment:
avoid to use tags as selectors except basic styles - `html, body, a, ol, ul, h1-h6`